src/fsharp/pars.fsy

```diff
@@ -2958,10 +2958,10 @@ atomicRationalConstant:
 
 constant: 
   | rawConstant
-    { $1 }
+    { $1, rhs parseState 1 }
 
   | rawConstant HIGH_PRECEDENCE_TYAPP measureTypeArg
-    { SynConst.Measure($1, $3) }
+    { SynConst.Measure($1, $3), rhs parseState 1 }
```

Comment:
it might just be that for constants with measures you need to return the range of the entire value here, instead of just the range of the raw value.

it might be worth pushing the range of just the constant into the `SynPat.Const` parameter of the `SynConst.Measure`?

like to get the full range of all parts of the measure you'd need the range of the constant, _and_ the range of the measure annotation.

Reply:
Thanks, Chet, that suggestion worked out and is also useful to have.